float64 demand
---
bool result
